File: day3/Quiz1/solution.py

def rangeArr(matrix):
    listMax = max(matrix)
    listMin = min(matrix)
    result = listMax-listMin-(len(matrix)-1)
    
    print(result)
    

if __name__ == "__main__":

    m = [1, 11, 8, 12]
    rangeArr(m)

File: day3/Quiz1/solution.java
import java.util.ArrayList;
import java.util.Arrays;

public class MyClass {
    public static int rangeArr(int[] list){
        int result = 0;
        int listMax = Arrays.stream(list).max().getAsInt();
        int listMin = Arrays.stream(list).min().getAsInt();    
        result = listMax - listMin -(list.length-1) ;

        return result;
        
    }
    public static void main(String args[]) {
        
        int [] pram = {1,2};
        System.out.println(rangeArr(pram));
    }
}
